tools/bazel

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
is this going to work with bb / bazelisk?

Reply:
Yep, I tested with both!